extensions/amp-access/0.1/test/test-amp-access-iframe.js

```diff
@@ -274,8 +275,8 @@ describes.realWin('AccessIframeAdapter', {
         });
       });
 
-      // TODO(dvoytenko, #14336): Fails due to console errors.
-      it.skip('should tolerate storage failures', () => {
+      it('should tolerate storage failures', () => {
+        const devErrorStub = sandbox.stub(dev(), 'error');
```

Comment:
It is no longer necessary to use this stub. You can use `allowConsoleError`, as described in #14406.